anyway/widgets/all_locations_widgets/seriously_injured_killed_in_bicycles_scooter_widget.py

```diff
@@ -0,0 +1,68 @@
+from typing import Dict
+
+from flask_babel import _
+from anyway.backend_constants import InjurySeverity, BackEndConstants as Constants
+from anyway.request_params import RequestParams
+from anyway.utilities import half_rounded_up
+from anyway.vehicle_type import VehicleType
+from anyway.widgets.all_locations_widgets.all_locations_widget import AllLocationsWidget
+from anyway.widgets.widget import register
+from anyway.widgets.widget_utils import get_involved_counts
+
+# noinspection PyProtectedMember
+TITLE = _("Number of severely injured or killed in bike, e-bike, or scooter accidents")
+
+
+@register
+class SeriouslyInjuredKilledInBicyclesScooterWidget(AllLocationsWidget):
+    name: str = "seriously_injured_killed_in_bicycles_scooter"
+    files = [__file__]
```

Comment:
Where are we using this variable?

Reply:
it is used in widget.py::register. seems like some hashing is done on the file and it's used to prevent widgets from registering twice